extensions/folder-bomb/package.json

```
@@ -0,0 +1,70 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "folder-bomb",
+  "title": "Folder Bomb",
+  "description": "Quickly take files out and move them to directory where parent folder is located.",
+  "icon": "extension-icon.png",
+  "author": "koinzhang",
+  "categories": [
+    "Productivity",
+    "System"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "move-file-out",
+      "title": "Move File out",
```

I'm not sure, but "Move File Out" doesn't really mean something to me that I understand what the command will do 😬 
Let's probably call the command:
```suggestion
      "title": "Move Selected Item to Parent Folder",
      "subtitle": "Folder Bomb",
```